roles/ceph-osd/tasks/bcache.yml

```diff
@@ -66,7 +72,12 @@
 - name: make-bcache -C <ssd device>
   command: make-bcache -C /dev/{{ ceph.bcache_ssd_device }}{{ ceph.disks|length + 1 }}
   ignore_errors: true
-  when: ok_to_deploy
+  when: ok_to_deploy and not bcache_ssd_is_nvme
```

Comment:
you will meet `bcache_ssd_is_nvme` undefined error here for SSD case, because we only define bcache_ssd_is_nvme when nvme is the cache device